build/repo.targets

```diff
@@ -0,0 +1,38 @@
+<Project>
+  <Import Project="common.props" />
+
+  <PropertyGroup>
+      <SiteExtensionProjectDirectory>$(RepositoryRoot)src\Microsoft.AspNetCore.AzureAppServices.TestBundle\</SiteExtensionProjectDirectory>
+      <PublishFeed Condition="$(PublishFeed) == ''">https://dotnet.myget.org/F/aspnetcore-ci-dev/</PublishFeed>
+      <DotnetChannel>master</DotnetChannel>
+      <DotnetVersion>coherent</DotnetVersion>
```

Comment:
I think we may need to tweak this. I tried building locally and master/coherent gave me 2.0.0-preview3-006889